frontend/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+interface CanvasPlacement {
+  name: string
+}
+
+interface Tool {
+  id: number,
+  name: string,
+  canvas_id: number,
+  logo_image: string | null,
```

Comment:
All of these This might look better with a user-defined type like 

`type Nullable<T> = T | null;`

I suppose the might be optional but probably easier for the API to just always return something. Though does it actually return a null now or just an empty string? I'd have to check unless you already have.

Reply:
I'm pretty sure it just returns `null` for now, and I believe I checked. The `Nullable<T>` is interesting, though I think it's basically just as long. I think I'll keep it this way for now, unless you object.